### ISSUE TITLE
Incorrect port detection

### ISSUE THREAD

**Issue by yangcao77** [devfile/api#1223]
## Which area this feature is related to?

/kind bug

### Which area this bug is related to?

/area alizer

### Go project

**Operating System and version:**

**Go Pkg Version:**

### Node.js project

**Operating System and version:**

**Node.js version:**

**Yarn version:**

**Project.json:**

### Web browser

**Operating System and version:**

**Browser name and version:**

## Bug Summary

**Describe the bug:**
Alizer returns the default port, but not the one set in Dockerfile

With a project with the default port set: 
```
// Listen and serve.
const PORT = process.env.PORT || 3000;
server.listen(PORT, () => {
  console.log(`App started on PORT ${PORT}.`);
});
```

and a Dockerfile sets the `env.PORT`
`ENV PORT 3001`

Alizer returns `ports: [3000]`. 

**To Reproduce:**

run Alizer `DetectComponents` against https://github.com/christianvogt/devfile-sample/tree/dockerfile/component

## Expected behavior

port `3001` should be detected, as in the code `process.env.PORT || 3000`, the env port has higher priority than the default port.

**Comment by thepetk** [comment]
The [PR](https://github.com/devfile/alizer/pull/23) for the proposed workaround is now under review :)

**Comment by thepetk** [comment]
The PR is still under review. Once it is approved we can merge it and [create a new release](https://github.com/devfile/alizer#release-process) for alizer (version: `v1.1.5`). Then we can inform the HAS team to pull latest alizer

**Comment by michael-valdron** [comment]
> The PR is still under review. Once it is approved we can merge it and [create a new release](https://github.com/devfile/alizer#release-process) for alizer (version: `v1.1.5`). Then we can inform the HAS team to pull latest alizer

Provided some review comments and suggestions: https://github.com/devfile/alizer/pull/23

**Comment by thepetk** [comment]
@yangcao77 the [PR](https://github.com/devfile/alizer/pull/23) is now merged and a new release of alizer (`v1.1.5`) is now published :) I've tested this case too and it works. In case is not working for you, feel free to reopen :)
